content/html/concepts/tags/terms/embed/embed.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: '<embed>'
+Description: 'The <embed> tag allows you to "Embed" a video, image, page and more.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Images'
+  - 'Video'
+  - 'HTML'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<embed>` tag allows you to embed content onto your page. It can be used to embed many types of content, however there are more specific tags such as `<video>`, `<img>` and `<iframe>`. It should only be used if there is no better option
```

Comment:
```suggestion
The `<embed>` tag allows you to embed content onto your page. It can be used to embed many types of content, however, there are more specific tags such as `<video>`, `<img>` and `<iframe>`. It should only be used if there is no better option.
```